docusign/authmetamodel.go

```diff
@@ -0,0 +1,39 @@
+package docusign
+
+import (
+	"github.com/amp-labs/connectors/common/paramsbuilder"
+)
+
+// Metadata fields that must be specified to initialize connector.
+var requiredMetadataFields = []string{ // nolint:gochecknoglobals
+	"server",
+}
+
+// AuthMetadataVars is a complete list of authentication metadata associated with connector.
+// This model serves as a documentation of map[string]string contents.
+type AuthMetadataVars struct {
+	ServerURL string
```

Comment:
For simplicity & maintainability, can we name the this field `Server` also?

Reply:
Yes, as it also turns out it is not a URL. Thats what I originaly thought from code, but that is a mistake.